pkgs/development/ocaml-modules/type_id/default.nix

```diff
@@ -0,0 +1,39 @@
+{ lib
+, buildDunePackage
+, fetchurl
+, alcotest
+, type_eq
+}:
+
+buildDunePackage rec {
+  pname = "type_id";
+  version = "0.0.1";
+
+  minimalOCamlVersion = "4.08.1";
+
+  src = fetchurl {
+    url = "https://github.com/skolemlabs/${pname}/releases/download/${version}/${pname}-${version}.tbz";
+    hash = "sha256-hmVAD9vgU1HLnB7d1TX17V+Alf5ZXmvQgd2nLHnLhDk=";
+  };
+
+  buidInputs = [
+    type_eq
+  ];
+
+
+  checkInputs = [
+    alcotest
+    type_eq
+  ];
+
+  doCheck = true;
+
+  meta = {
+    description = "Type identifiers, useful for runtime type-safe casting/coersions";
+    homepage = "https://github.com/skolemlabs/type_eq";
+    changelog = "https://github.com/skolemlabs/type_eq/blob/${version}/CHANGES.md";
```

Comment:
```suggestion
    homepage = "https://github.com/skolemlabs/type_id";
    changelog = "https://github.com/skolemlabs/type_id/blob/${version}/CHANGES.md";
```

Reply:
Whoops! I've done that a lot with this project :sigh:

Fixed!